include/constants/battle_config.h

```diff
@@ -62,9 +62,11 @@
 #define B_CRIT_MULTIPLIER           GEN_6 // In Gen6+, critical hits multiply damage by 1.5 instead of 2.
 #define B_EXP_CATCH                 GEN_6 // In Gen6+, Pokémon get experience from catching.
 #define B_TRAINER_EXP_MULTIPLIER    GEN_6 // In Gen7+, trainer battles no longer give a 1.5 multiplier to exp gain.
+#define B_SPLIT_EXP                 GEN_5 // In Gen6+, all participating mon get full experience.
+#define B_SCALED_EXP                GEN_6 // In Gen5 and Gen7+, experience is weighted by level difference.
```

Comment:
i think these should be a `TRUE` or `FALSE` setting rather than a generation setting (especially the scaled exp one, it's rather unintuitive that gen 5 enables, gen 6 disables, then gen 7 enables again)

Reply:
As discussed on Discord, this is probably not worth changing for consistency. Plus, using generational constants makes it more immediately obvious how to replicate a given gen's mechanics.